tasks/main.yml

```diff
@@ -48,6 +48,15 @@
   import_tasks: mcrypt.yml
   when: ansible_distribution_major_version|int > 9
 
+- name: Check if roundcube directory exists
+  stat:
+    path: "{{ roundcube_user_home }}/roundcubemail-{{ roundcube_version }}"
+  register: "roundcube_directory"
+
+- name: Set roundcube_directory_exists fact
```

Comment:
Why is it necessary to register an extra fact? Shouldn't it be possible to just check `roundcube_directory.stat.exists`?

Reply:
You are right.